docs: Update --emoji-reactions Documentation (#5327)
runatlantis/atlantis: runatlantis.io/docs/server-configuration.md

### `--emoji-reaction`

  ```bash
  atlantis server --emoji-reaction eyes
  # or
  ATLANTIS_EMOJI_REACTION=eyes
  ```

  The emoji reaction to use for marking processed comments. Currently supported on Azure DevOps, GitHub and GitLab. If not specified, Atlantis will not use an emoji reaction.
  Defaults to "" (empty string).

  ::: warning NOTE

>   Git provider API may be limiting the list of allowed emojis you can use
>   [Github](https://docs.github.com/en/rest/reactions/reactions?apiVersion=2022-11-28#about-reactions)
>   [Gitlab](https://gitlab.com/gitlab-org/gitlab/-/blob/master/fixtures/emojis/digests.json)

Reviewer: ```suggestion
  Each VCS provider supports a different lists of emojis:
  * [Github](https://docs.github.com/en/rest/reactions/reactions?apiVersion=2022-11-28#about-reactions)
  * [Gitlab](https://gitlab.com/gitlab-org/gitlab/-/blob/master/fixtures/emojis/digests.json)
```
